### PR TITLE
delete the cart

### PR DESCRIPTION
**Pull request by mohannadhanafi**
Relates #11 
- Delete the cart from database

### CODE REVIEW

File: src/views/cart.hbs [hunk 1]
```diff
@@ -9,7 +9,7 @@
         {{#each result }}
```
Comment (HemaSAli) [inline src/views/cart.hbs]: fix indentations

File: src/controllers/cart.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+  });
+};
+
+exports.delete = (req, res, next) => {
+  purchaseBox.destroy({ where: { id: req.body.boxId } }).then(() => {
+    res.status(200).send(JSON.stringify({ err: null, message: 'delete done' }));
+  }).catch((error) => {
+    next(error);
   });
 };
```
Comment (HemaSAli) [inline src/controllers/cart.js]: add new line

Comment (mohannadhanafi) [reply src/controllers/cart.js]: It's already exist

File: src/views/cart.hbs (hunk 1 above)
Comment (mohannadhanafi) [reply src/views/cart.hbs]: Done :+1:

**Pull request merged**